config/*

```diff
@@ -0,0 +1 @@
+/rails_conf/*
```

Comment:
is this used? seems like a symlink artifact from start script?

Reply:
Hmm, I don't know what git did here but I was trying to _delete_ the symlink... I'll see if I can remove that.